{
  "app_name": "TempApp",
  "error_fallback_title": "عفوًا، هناك خطأ ما",
  "error_fallback_message": "واجه التطبيق مشكلة ولم يتمكن من المتابعة. ونحن نعتذر عن أي إزعاج قد تسبب به هذا! اضغط على الزر أدناه لإعادة تشغيل التطبيق.",
  "restart_app": "أعد تشغيل التطبيق",
  "network_logs": "سجلات الشبكة",
  "login": "تسجيل الدخول",
  "username": "اسم المستخدم",
  "password": "كلمة المرور",
  "home": "الرئيسية",
  "notifications": "الاشعارات"
}
